src/client/Client.cc

```diff
@@ -12376,6 +12376,17 @@ size_t Client::_vxattrcb_mirror_info(Inode *in, char *val, size_t size)
                   in->xattrs["ceph.mirror.info.fs_id"].c_str());
 }
 
+size_t Client::_vxattrcb_cluster_fsid(Inode *in, char *val, size_t size)
+{
+  return snprintf(val, size, "%s", monclient->get_fsid().to_string().c_str());
+}
+
+size_t Client::_vxattrcb_client_id(Inode *in, char *val, size_t size)
+{
+  auto name = messenger->get_myname();
+  return snprintf(val, size, "%s.%ld", name.type_str(), name.num());
```

Comment:
Is this the same format we're going to use in the kernel?

Reply:
This is the output:

```
[root@lxbceph1 ~]# getfattr -n ceph.cluster_fsid /mnt/cephfs/file
# file: mnt/cephfs/file
ceph.cluster_fsid="b90ca47f-2352-4c5d-90f7-9248f1f1612f"

[root@lxbceph1 ~]# getfattr -n ceph.client_id /mnt/cephfs/file
# file: mnt/cephfs/file
ceph.client_id="client.4315"
```

The following is the kclient output:

```
[root@lxbceph1 kcephfs]# getfattr -n ceph.cluster_fsid file
# file: file
ceph.cluster_fsid="6ff21dc9-36b0-45a9-bec2-75aeaf0414cf" 

[root@lxbceph1 kcephfs]# getfattr -n ceph.client_id file
# file: file
ceph.client_id="client4358" 
```

There has an extra '.', I will remove it.

Reply:
Are we certain we want to use "client4358" as the format for the kernel? Those patches aren't merged yet, right? I think that format kind-of sucks but I realize that's how krbd presents it?

Reply:
> Are we certain we want to use "client4358" as the format for the kernel? Those patches aren't merged yet, right? I think that format kind-of sucks but I realize that's how krbd presents it?

Yeah, right, the kernel patch will stay consistent with the existing rbd attributes.

I think why the rbd formats it this way mainly dues to the debug file name will be:

`/sys/kernel/debug/ceph/6ff21dc9-36b0-45a9-bec2-75aeaf0414cf.client4358/`

Reply:
Yeah, I think it's valuable to maintain consistency with rbd on this.